number_of_robots: 1
#old: number_of_robots: 1
general_robot_name: "robot" # Name of every robot starts with this string. So if this is edited, robot_name, robot_namespace and robot_tf_prefix must be edited aswell 

plan_trajectory_action_name: "UR5ControllerPlanTrajectory"
execute_trajectory_action_name: "UR5ControllerExecuteTrajectory"
